rio_tiler/io/stac.py

```diff
@@ -19,6 +19,8 @@
 DEFAULT_VALID_TYPE = {
     "image/tiff; application=geotiff",
     "image/tiff; application=geotiff; profile=cloud-optimized",
+    "image/tiff; profile=cloud-optimized; application=geotiff",
+    pystac.MediaType.COG,
```

Comment:
Wait why are you adding both here? Aren't they identical?

Reply:
yes, they are but this ensures the compatibility even it changes in a future version of pystac.